src/facade/facade.cc

```diff
@@ -27,17 +27,19 @@ constexpr size_t kSizeConnStats = sizeof(ConnectionStats);
 
 ConnectionStats& ConnectionStats::operator+=(const ConnectionStats& o) {
   // To break this code deliberately if we add/remove a field to this struct.
-  static_assert(kSizeConnStats == 136u);
+  static_assert(kSizeConnStats == 152u);
 
   ADD(read_buf_capacity);
-  ADD(pipeline_cache_capacity);
+  ADD(dispatch_queue_entries);
+  ADD(dispatch_queue_bytes);
+  ADD(dispatch_queue_peak_bytes);
```

Comment:
interesting. global peak bytes are not the sum of peak bytes per connection or thread. I wonder if we can do this over estimation.

Reply:
True, the sum of peaks can be much higher even in modest usage scenarios. But then I have to find some place to store it... Will find one

Reply:
Fixed